Update code to follow new architecture plus updated conditions example (#6741)
umbraco/UmbracoDocs: 14/umbraco-cms/customizing/extending-overview/extension-types/condition.md

// Declare the Condition Configuration Type in the global UmbExtensionConditionConfigMap interface:
declare global {
    interface UmbExtensionConditionConfigMap {
        MyExtensionConditionConfig: MyExtensionConditionConfig;
    }
}
```

> The global declaration on the last five lines. Makes your Condition appear as a valid condition for manifests using the type `UmbExtensionManifest`. Also notice how the Condition Config Type alias has to match with the alias given when registrering the condition below.

Reviewer: ```suggestion
The global declaration on the last five lines makes your Condition appear valid for manifests using the type `UmbExtensionManifest`. Also, the Condition Config Type alias should match the alias given when registering the condition below.
```

}
```

> As can be seen in the code above, we did not make use of the configuration property `match` for our condition. We can do this by replacing the timeout with some other check:

Reviewer: ```suggestion
As shown in the code above, the configuration property `match` isn't used for our condition. We can do this by replacing the timeout with some other check:
```
